test/functional/apps/discover/_discover.ts

```diff
@@ -133,12 +135,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
         expect(Math.round(newDurationHours)).to.be(26);
 
-        await retry.waitFor('doc table to contain the right search result', async () => {
+        await retry.waitFor('doc table containing the documents of the brushed range', async () => {
           const rowData = await PageObjects.discover.getDocTableField(1);
-          log.debug(`The first timestamp value in doc table: ${rowData}`);
-          const dateParsed = Date.parse(rowData);
```

Comment:
So I'm sure this is the problem, the result of this should rely on the timezone of the env the test is started